src/note/models.py

```diff
@@ -21,15 +21,31 @@ def get_queryset(self):
         )
 
 
-class Note (models.Model):
+class NotePriority(models.Model):
+    DEFAULT_PRIORITY_ID = 1
```

Comment:
"Normal" is not always 1!

It's better to have a lambda function as the default that returns the priority=normal object.